doc/source/parameters/cfd/post_processing.rst

```diff
@@ -22,6 +22,7 @@ This subsection controls the post-processing other than the forces and torque on
   set verbosity                   = quiet
   set calculation frequency       = 1
   set output frequency            = 1
+  set smoothing                   = false
```

Comment:
I think an other name than smoothing would be better. Maybe continues derivatives or make results continuous?

Reply:
I was trying to fit the class name, but I understand the comment. How about `smoothed_output_fields` ?

Reply:
smoothed_output_fields has a nice ring to it.